Gemfile.lock

```diff
@@ -580,6 +580,3 @@ DEPENDENCIES
   weibo_2!
   wunderground (~> 1.2.0)
   xmpp4r (~> 0.5.6)
-
-BUNDLED WITH
-   1.10.5
```

Comment:
This 'feature' of bundler is really annoying, since we already have it in the Gemfile, do you mind updating bundler?